src/__tests__/__snapshots__/documenter.test.ts.snap

```diff
@@ -6805,31 +6851,6 @@ The event \`detail\` contains the current value of the component.",
       "optional": true,
       "type": "string",
     },
-    Object {
-      "description": "Adds \`aria-describedby\` to the component. If you're using this component within a form field,
-don't set this property because the form field component automatically sets it.
-Use this property if the component isn't surrounded by a form field, or you want to override the value
-automatically set by the form field (for example, if you have two components within a single form field).
-
-To use it correctly, define an ID for each element that you want to use as a description
-and set the property to a string of each ID separated by spaces (for example, \`\\"id1 id2 id3\\"\`).
-",
-      "name": "ariaDescribedby",
```

Comment:
why `ariaDescribedby` and `ariaLabelledby` got deleted?

Reply:
I am not sure tbh, I didn't do any changes on these APIs

Reply:
Switching from Omit to Exclude fixed it (excluding warning for file upload props since it doesn't make sense for this component to have this API)